script/alexa_locales.py

```diff
@@ -34,33 +50,43 @@ def run_script() -> None:
         if upstream_interface.count(".") == 1  # Skip sub-interfaces
     }
 
-    interfaces_missing = {}
-    interfaces_nok = {}
-    interfaces_ok = {}
+
+def compare_locales(upstream_locales: dict[str, set[str]]) -> tuple[list[str], list[str], list[str]]:
+    interfaces_missing = []
+    interfaces_nok = []
```

Comment:
What does `nok` mean?

Rename to `interfaces_with_missing_locales` or similar?

Reply:
In this case, `nok` is short for not ok.